tests/issues/issue-1022.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+return [
+    'issue-1022' => [
+        'ua' => 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.75 Safari/537.36 Google Favicon',
+        'properties' => [
+            'Comment' => 'Google Favicon',
+            'Browser' => 'Google Favicon',
+            'Browser_Type' => 'Bot/Crawler',
+            'Browser_Bits' => '64',
+            'Browser_Maker' => 'Google Inc',
+            'Browser_Modus' => 'unknown',
+            'Version' => '0.0',
+            'MajorVer' => '0',
+            'MinorVer' => '0',
+            'Platform' => 'Linux',
+            'Platform_Version' => 'unknown',
+            'Platform_Description' => 'Linux',
+            'Platform_Bits' => '64',
+            'Platform_Maker' => 'Linux Foundation',
+            'Alpha' => false,
+            'Beta' => false,
+            'Win16' => false,
+            'Win32' => false,
+            'Win64' => false,
+            'Frames' => true,
+            'IFrames' => true,
+            'Tables' => true,
+            'Cookies' => false,
+            'BackgroundSounds' => false,
+            'JavaScript' => true,
+            'VBScript' => false,
+            'JavaApplets' => false,
+            'ActiveXControls' => false,
+            'isMobileDevice' => false,
+            'isTablet' => false,
+            'isSyndicationReader' => false,
+            'Crawler' => true,
+            'isFake' => false,
+            'isAnonymized' => false,
+            'isModified' => false,
+            'CssVersion' => '0',
+            'AolVersion' => '0',
+            'Device_Name' => 'Linux Desktop',
+            'Device_Maker' => 'unknown',
+            'Device_Type' => 'Desktop',
+            'Device_Pointing_Method' => 'mouse',
+            'Device_Code_Name' => 'Linux Desktop',
+            'Device_Brand_Name' => 'unknown',
+            'RenderingEngine_Name' => 'unknown',
+            'RenderingEngine_Version' => 'unknown',
+            'RenderingEngine_Maker' => 'unknown',
+        ],
+        'lite' => false,
+        'standard' => true,
+        'full' => true,
+    ],
+    'issue-1022 (lite)' => [
```

Comment:
A couple of things about this approach.

1) Is it worth having a test for the Lite file for UAs like this one that aren't really what the UA is (but the lite file lacks the browser to properly detect it, which is expected). Does this create extra maintenance for little benefit?

2) Instead of creating a different test, would it be better to include these properties in the test above, perhaps "lite" and "standard" can be three different values (true/false/an array), like this?

```php
    'issue-1022' => [
        'ua' => 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.75 Safari/537.36 Google Favicon',
        'properties' => [
            'Comment' => 'Google Favicon',
            'Browser' => 'Google Favicon',
            'Browser_Type' => 'Bot/Crawler',
            'Browser_Bits' => '64',
            'Browser_Maker' => 'Google Inc',
            'Browser_Modus' => 'unknown',
            'Version' => '0.0',
            'MajorVer' => '0',
            'MinorVer' => '0',
            'Platform' => 'Linux',
            'Platform_Version' => 'unknown',
            'Platform_Description' => 'Linux',
            'Platform_Bits' => '64',
            'Platform_Maker' => 'Linux Foundation',
            'Alpha' => false,
            'Beta' => false,
            'Win16' => false,
            'Win32' => false,
            'Win64' => false,
            'Frames' => true,
            'IFrames' => true,
            'Tables' => true,
            'Cookies' => false,
            'BackgroundSounds' => false,
            'JavaScript' => true,
            'VBScript' => false,
            'JavaApplets' => false,
            'ActiveXControls' => false,
            'isMobileDevice' => false,
            'isTablet' => false,
            'isSyndicationReader' => false,
            'Crawler' => true,
            'isFake' => false,
            'isAnonymized' => false,
            'isModified' => false,
            'CssVersion' => '0',
            'AolVersion' => '0',
            'Device_Name' => 'Linux Desktop',
            'Device_Maker' => 'unknown',
            'Device_Type' => 'Desktop',
            'Device_Pointing_Method' => 'mouse',
            'Device_Code_Name' => 'Linux Desktop',
            'Device_Brand_Name' => 'unknown',
            'RenderingEngine_Name' => 'unknown',
            'RenderingEngine_Version' => 'unknown',
            'RenderingEngine_Maker' => 'unknown',
        ],
        'lite' => [
            'properties' => [
                'Comment' => 'Chrome 49.0',
                'Browser' => 'Chrome',
                'Version' => '49.0',
                'Platform' => 'Linux',
                'isMobileDevice' => false,
                'isTablet' => false,
                'Device_Type' => 'Desktop',
            ],
        ],
        'standard' => true,
    ],
```

`true` would imply that the properties should match what is defined, `false` would imply not to test this UA against the file type and an array of properties would imply that the UA should be tested, but the properties should be as defined (which is different from the defined props).  I think if you went with an approach like this, the `full` key could be removed, as every UA in the fixtures should be tested by the full file right (I believe the `full` key is being added here because you're duplicating some UAs in the fixtures in order to specify how they should differ for the lite and standard files when they do)?